mmv1/templates/terraform/custom_flatten/bigtable_app_profile_routing.tmpl

```diff
@@ -25,5 +25,10 @@ func flatten{{$.GetPrefix}}{{$.TitlelizeProperty}}(v interface{}, d *schema.Reso
 		}
 	}
 
+	if v.(map[string]interface{})["rowAffinity"] != nil {
```

Comment:
Should this be a bit higher up? Maybe in line 17? Is it possible that we have no clusterIds , but still want row_affinity? or maybe the other if statement?